tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/AlfrescoStackInitializer.java

```diff
@@ -0,0 +1,139 @@
+
+        GenericContainer sfs = new GenericContainer("alfresco/alfresco-shared-file-store:latest")
+                                       .withNetwork(network)
+                                       .withNetworkAliases("shared-file-store")
+                                       .withEnv("JAVA_OPTS", "-Xms256m -Xmx512m")
+                                       .withEnv("scheduler.content.age.millis", "86400000")
+                                       .withEnv("scheduler.cleanup.interval", "86400000");
+
+        PostgreSQLContainer postgres = (PostgreSQLContainer) new PostgreSQLContainer("postgres:13.1")
+                                                                     .withPassword("alfresco")
+                                                                     .withUsername("alfresco")
+                                                                     .withDatabaseName("alfresco")
+                                                                     .withNetwork(network)
+                                                                     .withNetworkAliases("postgres");
+
+        GenericContainer activemq = new GenericContainer("alfresco/alfresco-activemq:5.15.8")
+                                            .withNetwork(network)
+                                            .withNetworkAliases("activemq")
+                                            .waitingFor(Wait.forListeningPort())
+                                            .withExposedPorts(61616, 8161, 5672, 61613);
+
+        elasticsearch = new ElasticsearchContainer("docker.elastic.co/elasticsearch/elasticsearch:7.10.1")
+                                .withNetwork(network)
+                                .withNetworkAliases("elasticsearch")
+                                .withExposedPorts(9200)
+                                .withEnv("xpack.security.enabled", "false")
+                                .withEnv("discovery.type", "single-node");
+
+        liveIndexer = new GenericContainer("quay.io/alfresco/alfresco-elasticsearch-live-indexing:" + ES_CONNECTOR_TAG)
+                              .withNetwork(network)
+                              .withNetworkAliases("live-indexing")
+                              .withEnv("ELASTICSEARCH_INDEXNAME", "custom-alfresco-index")
+                              .withEnv("SPRING_ELASTICSEARCH_REST_URIS", "http://elasticsearch:9200")
+                              .withEnv("SPRING_ACTIVEMQ_BROKERURL", "nio://activemq:61616")
+                              .withEnv("ALFRESCO_SHAREDFILESTORE_BASEURL", "http://shared-file-store:8099/alfresco/api/-default-/private/sfs/versions/1/file/");
+
+        CompletableFuture<Void> start = Startables.deepStart(elasticsearch, postgres, alfresco, activemq, transformCore, transformRouter, liveIndexer, sfs);
+
+        try
+        {
+            start.get();
+        } catch (Exception e)
```

Comment:
[Minor] Formatting rules for repo projects have braces on their own lines (except in lambdas).

Reply:
done